app/services/encryption/encryptors/attribute_encryptor.rb

```diff
@@ -0,0 +1,67 @@
+module Encryption
+  module Encryptors
+    class AttributeEncryptor
+      def encrypt(plaintext)
+        user_access_key = self.class.load_or_init_user_access_key(
+          key: current_key, cost: current_cost
+        )
+        UserAccessKeyEncryptor.new(user_access_key).encrypt(plaintext)
+      end
+
+      def decrypt(ciphertext)
+        encryption_keys_with_cost.each do |key_with_cost|
+          key = key_with_cost.fetch(:key)
+          cost = key_with_cost.fetch(:cost)
+          result = try_decrypt(ciphertext, key: key, cost: cost)
+          return result unless result.nil?
+        end
+        raise Pii::EncryptionError, 'unable to decrypt attribute with any key'
+      end
+
+      def stale?
+        return false if stale.nil?
```

Comment:
I'm not sure about the value of this since `stale` is falsey if it's `nil`. If `stale` isn't `nil`, then it's not coerced into a boolean.